### PR TITLE
Fix missing spacings in source code

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
I pushed a patch staging on mailing-list too, as it is about the same.

### CODE REVIEW

File: crypto/bn/bn_lcl.h [hunk 1]
```diff
@@ -157,18 +157,18 @@ int RAND_pseudo_bytes(unsigned char *buf, int num);
...
-                         * wouldn't be constructed with top!=dmax. */ \
-                        BN_ULONG *_not_const; \
-                        memcpy(&_not_const, &_bnum1->d, sizeof(_not_const)); \
-                        RAND_bytes(&_tmp_char, 1); /* Debug only - safe to ignore error return */\
-                        memset(_not_const + _bnum1->top, _tmp_char, \
-                                sizeof(*_not_const) * (_bnum1->dmax - _bnum1->top)); \
-                } \
+            const BIGNUM *_bnum1 = (a); \
+            if (_bnum1->top < _bnum1->dmax) { \
+                    unsigned char _tmp_char; \
```
Comment (richsalz) [inline crypto/bn/bn_lcl.h]: why is this indented eight, not four?

File: crypto/include/internal/evp_int.h [hunk 2]
```diff
@@ -193,15 +193,15 @@ static int cname##_cfb##cbits##_cipher(EVP_CIPHER_CTX *ctx, unsigned char *out,
         if (cbits==1)  chunk>>=3;\
         if (inl<chunk) chunk=inl;\
         while(inl && inl>=chunk)\
-            {\
+        {\
```
Comment (richsalz) [inline crypto/include/internal/evp_int.h]: this whole block of code is wrongly formatted.  maybe not for this MR, tho.

**Comment by richsalz** [comment]
The comma-space fixes in the SYNOPSIS sections are covered by https://gitlab.openssl.org/openssl/openssl/merge_requests/3039 (which is team-internal)

**Comment by FdaSilvaYY** [comment]
OK, once you will push this MR, my rebase will remove these SYNOPSIS changes.

**Comment by FdaSilvaYY** [comment]
Rebased !

**Comment by richsalz** [comment]
There is another internal MR that still needs review and posting…

File: crypto/bio/bss_mem.c [hunk 3]
```diff
@@ -168,7 +168,7 @@ static int mem_buf_sync(BIO *b)
     if((b != NULL) && (b->init) && (b->ptr != NULL)) {
```
Comment (richsalz) [inline crypto/bio/bss_mem.c]: you missed one :)  and if you touch it, please remove the extra internal parens.

**Comment by richsalz** [comment]
okay, rebase again :)  then i can merge this.

**Comment by FdaSilvaYY** [comment]
Done.

File: crypto/include/internal/evp_int.h [hunk 4]
```diff
@@ -189,20 +189,25 @@ static int cname##_cbc_cipher(EVP_CIPHER_CTX *ctx, unsigned char *out, const uns
...
+        int num = EVP_CIPHER_CTX_num(ctx);\
+        cprefix##_cfb##cbits##_encrypt(in, out, (long) \
+            ((cbits == 1) \
+                && !EVP_CIPHER_CTX_test_flags(ctx, EVP_CIPH_FLAG_LENGTH_BITS) \
+                ? inl*8 : inl), \
+            &EVP_C_DATA(kstruct, ctx)->ksched, EVP_CIPHER_CTX_iv_noconst(ctx),\
+            &num, EVP_CIPHER_CTX_encrypting(ctx));\
+        EVP_CIPHER_CTX_set_num(ctx, num);\
+        inl -= chunk;\
+        in  += chunk;\
```
Comment (mattcaswell) [inline crypto/include/internal/evp_int.h]: Too many spaces here

File: doc/ssl/SSL_CTX_set_generate_session_id.pod [hunk 5]
```diff
@@ -90,25 +90,25 @@ server id given, and will fill the rest with pseudo random bytes:
...
-                      strlen(session_id_prefix) : *id_len);
-              }
-      while(SSL_has_matching_session_id(ssl, id, *id_len) &&
+      do {
+          RAND_pseudo_bytes(id, *id_len);
+          /* Prefix the session_id with the required prefix. NB: If our
+           * prefix is too long, clip it - but there will be worse effects
+           * anyway, eg. the server could only possibly create 1 session
+           * ID (ie. the prefix!) so all future session negotiations will
+           * fail due to conflicts. */
```
Comment (mattcaswell) [inline doc/ssl/SSL_CTX_set_generate_session_id.pod]: The style is to have have multi-line comments start and end with `\*` and `*\` on a line on their own.

Comment (richsalz) [inline doc/ssl/SSL_CTX_set_generate_session_id.pod]: This isn't fixing the docs to fully do openssl style.  I'd say that 's a separate MR.

File: ssl/record/rec_layer_s3.c [hunk 6]
```diff
@@ -1186,8 +1186,8 @@ int ssl3_read_bytes(SSL *s, int type, int *recvd_type, unsigned char *buf,
         goto f_err;
     }
 
-    if(s->method->version == TLS_ANY_VERSION
-            && (s->server || rr->type != SSL3_RT_ALERT)) {
+    if (s->method->version == TLS_ANY_VERSION
+        && (s->server || rr->type != SSL3_RT_ALERT)) {
```
Comment (mattcaswell) [inline ssl/record/rec_layer_s3.c]: The additional indent on the `&&` is intentional to distinguish it from the body of the block.

Comment (richsalz) [inline ssl/record/rec_layer_s3.c]: About which our style isn't clear, right?

File: doc/ssl/SSL_CTX_set_generate_session_id.pod (hunk 5 above)
Comment (mattcaswell) [inline doc/ssl/SSL_CTX_set_generate_session_id.pod]: No...but while its being touched anyway we should fix it. No need to go through the rest of the docs.

File: ssl/record/rec_layer_s3.c (hunk 6 above)
Comment (mattcaswell) [inline ssl/record/rec_layer_s3.c]: True. Not a biggie.

**Comment by richsalz** [comment]
So i'll wait for you to finish @mattcaswell and let @FdaSilvaYY address your issues, then I'll reconfirm my plus-one and merge.

**Comment by mattcaswell** [comment]
+1

**Comment by richsalz** [comment]
thanks for the keen-eyed MR and review :)  pushed.

**Pull request closed without merging**